spec/features/admin_deletes_hotspot_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+feature 'Admin deletes hotspot' do
+  scenario do
+    hotspot = create(:hotspot)
+    visit edit_hotspot_path(hotspot)
+    
+    click_link 'Delete'
```

Comment:
need an empty line here (since `click_link` is the exercise section and the lines below are the verification section) https://robots.thoughtbot.com/four-phase-test